conf/maestro-conf/edge-config-dell5000-demo.yaml

```diff
@@ -80,13 +73,15 @@ mdns:
 symphony:
 # symphony system management APIs
     # defaults to 10:
+    disable_sys_stats: true
     sys_stats_count_threshold: 15     # send if you have 15 or more stats queued
     sys_stats_time_threshold: 120000  # every 120 seconds send stuff, no matter what
-    client_cert: "{{ARCH_CLIENT_CERT_PEM}}"
-    client_key: "{{ARCH_CLIENT_KEY_PEM}}"
-    host: "{{ARCH_GW_SERVICES_RESRC}}"
-    url_logs: "{{ARCH_GW_SERVICES_URL}}/relay-logs/logs"
-    url_stats: "{{ARCH_GW_SERVICES_URL}}/relay-stats/stats_obj"
+    #client_cert: "{{ARCH_CLIENT_CERT_PEM}}"
+    #client_key: "{{ARCH_CLIENT_KEY_PEM}}"
+    no_tls: true
+    host: "gateways.local"
```

Comment:
I do not believe these should be hard-coded. The mustache template that exists already is there specifically to parse keys from identity.json.

Reply:
@zhucebuliaolongchuan Isn't fp-edge also hardcoded to listen for requests from "gateways.local"
If so, isn't it okay if this is hardcoded here.
if not, @mray190  should we  get them from identity.json?

Reply:
Still think we should fix the mustache to resolve to localhost or gateways.local. But also now realize that gateways.local points to localhost

Reply:
I am sorry but I am not sure what exactly a "mustache" is. It would be great if you can explain what you mean by that.

Reply:
@psppalash we could do it I guess, but I don't think it is that easy to generate such a field for identity.json right?

Reply:
@psppalash  For example - "{{ARCH_CLIENT_KEY_PEM}}" is a mustache which gets populated by maestro. Similarly there are var_defs in maestro. Better solution will be to create a new var_def here - https://github.com/armPelionEdge/edge-utils/blob/master/conf/maestro-conf/edge-config-dell5000-demo.yaml#L17

Reply:
some of those mustaches are defined in the var_defs section, for example SNAP_DATA, but others are not, for example ARCH_SERIAL_NUMBER.  For those variables that are not defined in var_defs, how are they resolved?

Reply:
Anything prefixed with "ARCH_" comes from platform reader. I think now we are using "fsonly" platform reader.